### DISCUSSION TITLE
框架是不是更新太慢了？

### DISCUSSION THREAD

**Discussion by langziyang** [DouyinFE/semi-design#2187] [General]
useNetwork.jsx:57 
 Warning: ReactDOM.render is no longer supported in React 18. Use createRoot instead. Until you switch to the new API, your app will behave as if it's running React 17. Learn more:
···
Toast.success(() => ({
                        content: resp['hydra:member'] || successMessage,
                        duration: 5,
                        stack: true,
                    }))
···

**Comment by pointhalo** [comment]
你说的这个问题，由于目前我们需要兼顾 16、17、18的用户群。没有办法移除这个使用方式。详细技术方案限制解释可见 https://github.com/DouyinFE/semi-design/issues/938
